src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.DefaultProxyCredentials.cs

```diff
@@ -42,87 +43,90 @@ public void SetGet_Roundtrips()
             }
         }
 
-        [ActiveIssue(23702, TargetFrameworkMonikers.NetFramework)]
         [ActiveIssue(20010, TargetFrameworkMonikers.Uap)]
-        [OuterLoop] // TODO: Issue #11345
         [Fact]
         public async Task ProxyExplicitlyProvided_DefaultCredentials_Ignored()
         {
-            int port;
-            Task<LoopbackGetRequestHttpProxy.ProxyResult> proxyTask = LoopbackGetRequestHttpProxy.StartAsync(out port, requireAuth: true, expectCreds: true);
-            Uri proxyUrl = new Uri($"http://localhost:{port}");
-
             var rightCreds = new NetworkCredential("rightusername", "rightpassword");
             var wrongCreds = new NetworkCredential("wrongusername", "wrongpassword");
+            string expectCreds = "Basic " + Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes($"{rightCreds.UserName}:{rightCreds.Password}"));
 
-            using (HttpClientHandler handler = CreateHttpClientHandler())
-            using (var client = new HttpClient(handler))
+            await LoopbackServer.CreateServerAsync(async (proxyServer, proxyUrl) =>
             {
-                handler.Proxy = new UseSpecifiedUriWebProxy(proxyUrl, rightCreds);
-                handler.DefaultProxyCredentials = wrongCreds;
-
-                Task<HttpResponseMessage> responseTask = client.GetAsync(Configuration.Http.RemoteEchoServer);
-                Task<string> responseStringTask = responseTask.ContinueWith(t =>
+                using (HttpClientHandler handler = CreateHttpClientHandler())
+                using (var client = new HttpClient(handler))
                 {
-                    using (t.Result) return t.Result.Content.ReadAsStringAsync();
-                }, TaskScheduler.Default).Unwrap();
-                await (new Task[] { proxyTask, responseTask, responseStringTask }).WhenAllOrAnyFailed();
+                    handler.Proxy = new UseSpecifiedUriWebProxy(proxyUrl, rightCreds);
+                    handler.DefaultProxyCredentials = wrongCreds;
 
-                TestHelper.VerifyResponseBody(responseStringTask.Result, responseTask.Result.Content.Headers.ContentMD5, false, null);
-                Assert.Equal(Encoding.ASCII.GetString(proxyTask.Result.ResponseContent), responseStringTask.Result);
-
-                string expectedAuth = $"{rightCreds.UserName}:{rightCreds.Password}";
-                Assert.Equal(expectedAuth, proxyTask.Result.AuthenticationHeaderValue);
-            }
+                    Task<HttpResponseMessage> responseTask = client.GetAsync(Configuration.Http.RemoteEchoServer);
```

Comment:
Are you actually going to connect to the RemoteEchoServer here?  If not, it's clearer to pass a dummy value like "http://notatrealserver.com/" or whatever.